### PR TITLE
minimal support for raw vectors.

### PR DESCRIPTION
**Pull request by romainfrancois**
cleaner version of #345
I manually applied the changes from the broken PR to the current master

### CODE REVIEW

**Comment by romainfrancois** [comment]
🤔 the failures seem related to the `.default` argument in `map`: 

https://travis-ci.org/tidyverse/purrr/jobs/337610412#L1150

**Comment by hadley** [comment]
Yeah, that was my fault - can you please merge/rebase from master?

**Comment by romainfrancois** [comment]
So I have 

```
$ git remote -v
origin	https://github.com/romainfrancois/purrr.git (fetch)
origin	https://github.com/romainfrancois/purrr.git (push)
upstream	https://github.com/tidyverse/purrr.git (fetch)
upstream	https://github.com/tidyverse/purrr.git (push)
```

and then I've done 

```
git fetch upstream
git merge upstream/master
git push
```

I might have to properly learn about rebase some day.

**Comment by romainfrancois** [comment]
I guess I was close with my rebase attempts and all I needed really was the `-f` as described in https://gist.github.com/ravibhure/a7e0918ff4937c9ea1c456698dcd58aa

File: DESCRIPTION [hunk 1]
```diff
@@ -25,4 +25,5 @@ Suggests:
 VignetteBuilder: knitr
 LazyData: true
 Roxygen: list(markdown = TRUE)
-RoxygenNote: 6.0.1
```
Comment (hadley) [inline DESCRIPTION]: Can you please use the released version of roxygen2?

**Comment by hadley** [comment]
Can you please also add a bullet to NEWS?

**Comment by romainfrancois** [comment]
I think I managed to do a clean rebase this time 🎉

**Comment by hadley** [comment]
Looks good!

Can you please add a bullet to NEWS? It should briefly describe the change (starting with name of the function), and crediting yourself with `(@yourname, #issuenumber)`.

**Comment by romainfrancois** [comment]
Done. Does it make sense to do the same for complex  for completeness or is it a maintenance burden down the line ?

**Comment by hadley** [comment]
I don't see the benefit to doing it until someone has expressed a need.

**Comment by hadley** [comment]
Thanks!

**Pull request merged**